specs/Standards/Wallets/InjectedWallets.md

```diff
@@ -0,0 +1,325 @@
+- Access to multiple accounts allow dApps more freedom to improve UX as users can seamlessly switch between accounts.
+- Aligns with WalletConnect via the [Bridge Wallet Standard](./BridgeWallets.md).
+
+### Storage of key pairs for FunctionCall access keys in dApp context vs. wallet context
+
+- NEAR's unique concept of `FunctionCall` access keys allow for the concept of 'signing in' to a dApp using your wallet.  'Signing In' to a dApp is accomplished by adding `FunctionCall` type access key that the dApp owns to the account that the user is logging in as.
+- Once a user has 'signed in' to a dApp, the dApp can then use the keypair that it owns to execute transactions without having to prompt the user to route and approve those transactions through their wallet. 
+- `FunctionCall` access keys have a limited quota that can only be used to pay for gas fees (typically 0.25 NEAR) and can further be restricted to only be allowed to call *specific methods* on one **specific** smart contract.
+- This allows for an ideal user experience for dApps that require small gas-only transactions regularly while in use.  Those transactions can be done without interrupting the user experience by requiring them to be approved through their wallet.  A great example of this is evident in gaming use-cases -- take a gaming dApp where some interactions the user makes must write to the blockchain as they do common actions in the game world.  Without the 'sign in' concept that provides the dApp with its own limited usage key, the user might be constantly interrupted by needing to approve transactions on their wallet as they perform common actions. If a player has their account secured with a ledger, the gameplay experience would be constantly interrupted by prompts to approve transactions on their ledger device!  With the 'sign in' concept, the user will only intermittently need to approve transactions to re-sign-in, when the quota that they approved for gas usage during their last login has been used up.
+- Generally, it is recommended to only keep `FullAccess` keys in wallet scope and hidden from the dApp consumer. `FunctionCall` type keys should be generated and owned by the dApp, and requested to be added using the `signIn` method. They should **not** be 'hidden' inside the wallet in the way that `FullAccess` type keys are.
+
+## Specification
+
+Injected wallets are typically browser extensions that implement the `Wallet` API (see below).  References to the currently available wallets are tracked on the `window` object. To avoid namespace collisions and easily detect when they're available, wallets must mount under their own key of the object `window.near` (e.g. `window.near.sender`).
+**NOTE: Do not replace the entire `window.near` object with your wallet implementation, or add any objects as properties of the `window.near` object that do not conform to the Injected Wallet Standard**
```

Comment:
What's advantage and disadvantage of mounting the `Wallet` object at `window.near` here?

I think developers who're familiar with Ethereum may prefer to use `window.near` as the root of the Wallet API, as long as they can switch among the providers.